src/transformers/models/deformable_detr/modeling_deformable_detr.py

```diff
@@ -1186,13 +1202,16 @@ def forward(
             position_embeddings (`torch.FloatTensor` of shape `(batch_size, num_queries, hidden_size)`, *optional*):
                 Position embeddings that are added to the queries and keys in each self-attention layer.
 
-            reference_points
+            reference_points (`torch.FloatTensor` of shape `(batch_size, num_queries, 4)` is `as_two_stage` else `(batch_size, num_queries, 2)` or , *optional*):
```

Comment:
```suggestion
            reference_points (`torch.FloatTensor` of shape `(batch_size, num_queries, 4)` if `config.two_stage` else `(batch_size, num_queries, 2)` or , *optional*):
```
I guess you mean this?
```suggestion
            reference_points (`torch.FloatTensor` of shape `(batch_size, num_queries, 4)` is `as_two_stage` else `(batch_size, num_queries, 2)` or , *optional*):
```